### DISCUSSION TITLE
Simplify object member specification

### DISCUSSION THREAD

**Discussion by schaumb** [stephenberry/glaze#629] [Ideas]
Currently, if I have a custom struct (which is not aggregate) I need to specify all member pointers to the `glz::object` parameter with their names ([like here](https://github.com/stephenberry/glaze/blob/main/tests/lib_test/interface.hpp#L27-L32)). 

It can be user-friendlier if the name is deduced from the member pointer, and it does not need to be duplicated.

I think about something:
`object<&T::x, &T::y, ...>` which iterates all template arguments (member pointer or null parameterized member function) and gets their names with the member name getter technique  (example: [the definition](https://github.com/vizzuhq/vizzu-lib/blob/259851a44632a89f1146035902d4563b460fefa9/src/chart/main/style.h#L61-L68) and the [parser](https://github.com/vizzuhq/vizzu-lib/blob/main/src/base/refl/auto_struct.h#L421-L423) is almost same as with the member reflection.)

**Comment by stephenberry** [comment]
Your request is already supported. You can just write: glz::object(&T::x, ...)

Glaze will reflect your member pointer names. See the latest README for more explanation.

**Comment by schaumb** [comment]
Thanks! I missed that part!
